docs/features/configuration.rst

```diff
@@ -132,6 +132,16 @@ You can also give Ocelot a specific path to look in for the configuration files
 
 Ocelot needs the HostingEnvironment so it knows to exclude anything environment specific from the algorithm. 
 
+Merging configuration files to memory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default Ocelot writes the merged configuration back to disk as ``ocelot.json`` and add that file to config.
+If your server don't have write permissions to your configuration folder, you can instruct Ocelot to use the merged configuration directly from memory instead.

```

Comment:
We need to explain and uncover these user scenarios!
We need to show that merging config to a memory is required mostly in all virtual environments like cloud providers (AWS, Azure, GCP).